src/System.Runtime.Extensions/tests/System/StringComparer.netcoreapp1.1.cs

```diff
@@ -0,0 +1,33 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Globalization;
+using Xunit;
+
+namespace System.Tests
+{
+    public static partial class MiscStringComparerTests
+    {
+        public static object[][] FromComparison_TestData()
+        {
+            return new object[][] {
```

Comment:
My suggestion was to turn this into a static read only field, instead of a method. Then you won't need the return statement.

```c#
public static readonly object[][] FromComparison_TestData =
{
    new object[] { StringComparison.CurrentCulture, StringComparer.CurrentCulture },
    new object[] { StringComparison.CurrentCultureIgnoreCase, StringComparer.CurrentCultureIgnoreCase },
    // etc.
};
```